formatting_NEON_microbial/1. merging and cleaning october 16S files.r

```diff
@@ -0,0 +1,109 @@
+#Processing 16S bacterial OTU tables with taxonomy and mapping files sent by Lee Stanish October, 2017
+rm(list=ls())
+library(biomformat)
+library(nneo)
+#subtring function to pull from the right edge of a character entry.
+substrRight <- function(x, n){
+  substr(x, nchar(x)-n+1, nchar(x))
+}
+
+#set output paths for cleaned otu, map, and tax files.
+otu.out <- '/fs/data3/caverill/NEFI_microbial/map_otu/16S_otu_clean.rds'
```

Comment:
I'd recommend defining the folder path as a separate variable, and then reusing that variable, rather than always hardcoding it. It will make the code much more portable and generalizable (e.g. if you got an updated version of the data that you wanted to process in a different folder)